jina/docker/hubio.py

```diff
@@ -467,29 +468,35 @@ def _test_build(image,  # type docker image object
                     test_level: 'BuildTestLevel',
                     config_yaml_path: str,
                     timeout_ready: int,
-                    daemon_arg: bool):
+                    daemon_arg: bool,
+                    logger: 'JinaLogger'):
         p_names = []
         failed_levels = []
-
+        logger.info(f'run tests using test level {test_level}')
         # test uses at executor level
         if test_level >= BuildTestLevel.EXECUTOR:
+            logger.info(f'test executor')
```

Comment:
I think some better messages can be recorded. Testing with test-level blah blah ..